apps/system/js/system_messages.js

```diff
@@ -1,8 +1,17 @@
-/* global BaseModule */
+/* global BaseModule, BroadcastChannel, Service */
 'use strict';
 
 (function() {
-  // System Messages placeholder
+  var BASE_URL = 'chrome://gaia/content';
+  var MESSAGES = {
+    'bluetooth-dialer-command': '/dialer/index.html#keyboard-view',
+    'headset-button': '/dialer/index.html#keyboard-view',
+    'notification': '/dialer/index.html#keyboard-view',
```

Comment:
Right now the dialer is the only app that will post notifications but we'll need a way to figure out how to steer them towards the right app once we get more apps working into chrome. So this works for now but let's keep in mind that it won't cut it once we have more than one app excepting one message.